### PR TITLE
Avatar

### PR DESCRIPTION
**Pull request by rubenmoya**
#43
----

- [x] Implementation
- [x] Design Aproval
- [x] UI Test
- [x] CR

### CODE REVIEW

**Comment by piensaenpixel** [comment]
I think this is broken
image

File: packages/icons/dist/webfont.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
+@font-face {
+    font-family: webfont;
+    src: url("./webfont.eot");
+    src: url("./webfont.eot?#iefix") format("embedded-opentype"), url("./webfont.woff2") format("woff2"), url("./webfont.woff") format("woff"), url("./webfont.ttf") format("truetype"), url("./webfont.svg#webfont") format("svg");
+    font-style: normal;
+    font-weight: 400;
+}
+
+.webfont {
```
Comment (IagoLast) [inline packages/icons/dist/webfont.css]: webfont?

Comment (rubenmoya) [reply packages/icons/dist/webfont.css]: well, someone changed the `.gitignore` :P

Comment (IagoLast) [reply packages/icons/dist/webfont.css]: xD

**Pull request merged**